lightning/src/ln/channelmanager.rs

```diff
@@ -1680,7 +1680,12 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 						features: InitFeatures::empty(),
 						unspendable_punishment_reserve: to_remote_reserve_satoshis,
 						forwarding_info: channel.counterparty_forwarding_info(),
-						outbound_htlc_minimum_msat: Some(channel.get_counterparty_htlc_minimum_msat()),
+						// Ensures that we have actually received the `htlc_minimum_msat` value
+						// from the counterparty through the `OpenChannel` or `AcceptChannel`
+						// message (as they are always the first message from the counterparty).
+						// Else `Channel::get_counterparty_htlc_minimum_msat` could return the
+						// default `0` value set by `Channel::new_outbound`.
+						outbound_htlc_minimum_msat: channel.have_received_message().then(|| channel.get_counterparty_htlc_minimum_msat()),
```

Comment:
Won't be able to use `then` because our MSRV is 1.41.1. Will have to use an if-else expression.

Reply:
Oh ok thanks! I'll update immediately